### PR TITLE
Update elastic-apm-node version.

### PR DESCRIPTION
**Pull request by kbader-godaddy**
## Summary

We are seeing a bad engine console warning during `npm ci`.

Screenshot 2023-09-29 at 2 24 38 PM

## Changelog

Update elastic-apm-node version.
## Test Plan

CICD passes.

### CODE REVIEW

File: packages/gasket-nextjs/package.json [hunk 1]
```diff
@@ -43,7 +43,9 @@
   "homepage": "https://github.com/godaddy/gasket/tree/main/packages/gasket-nextjs",
   "dependencies": {
     "@gasket/data": "^6.41.1",
-    "htmlescape": "^1.1.1"
+    "htmlescape": "^1.1.1",
+    "prop-types": "^15.8.1",
+    "react": "^18.2.0"
```
Comment (rmarkins-godaddy) [inline packages/gasket-nextjs/package.json:48]: Why did these get moved?

Comment (jpage-godaddy) [inline packages/gasket-nextjs/package.json:48]: Shouldn't this be a `peerDependency` instead to help avoid duplicated react instances?

File: packages/gasket-nextjs/package.json [hunk 2]
```diff
@@ -66,7 +68,6 @@
     "mock-require": "^3.0.3",
     "next": "^12.0.8",
     "nyc": "^15.1.0",
-    "react": "^18.0.0",
```
Comment (jpage-godaddy) [inline packages/gasket-nextjs/package.json:69]: I'd leave this in `devDependencies` and pair it with an entry in `peerDependencies`.

**Review by jpage-godaddy — APPROVED** [review]
Elastic APM change is a definite 👍

File: packages/gasket-nextjs/package.json (hunk 1 above)
Comment (agerard-godaddy) [inline packages/gasket-nextjs/package.json:48]: `react` should continue to be a devDep to allow control of version at the app level. I would say same `prop-types`.

**Review by agerard-godaddy — CHANGES_REQUESTED** [review]

File: packages/gasket-nextjs/package.json (hunk 2 above)
Comment (kbader-godaddy) [reply packages/gasket-nextjs/package.json:69]: Should I do the same for prop-types?

File: packages/gasket-nextjs/package.json (hunk 1 above)
Comment (rmarkins-godaddy) [reply packages/gasket-nextjs/package.json:48]: 100% agree those should both stay as devDep

Comment (kbader-godaddy) [reply packages/gasket-nextjs/package.json]: Great catch! This actually needed to become move to `peerDependencies`. Thanks for the review!

Comment (kbader-godaddy) [reply packages/gasket-nextjs/package.json]: Yes exactly! Just pushed the update. Thanks for the review!

Comment (kbader-godaddy) [reply packages/gasket-nextjs/package.json]: Awesome. Updates have been made. Appreciate the reviews!

**Review by agerard-godaddy — APPROVED** [review]

**Pull request merged**